src/extension/manifest.json

```diff
@@ -12,8 +12,7 @@
     "512": "/assets/icon-512.png"
   },
   "background": {
```

Comment:
Chrome and FireFox seem to conflict here

- [chrome](https://developer.chrome.com/docs/extensions/mv3/mv3-migration/#man-sw)

while FF linter sais that `service_worker` is not allowed